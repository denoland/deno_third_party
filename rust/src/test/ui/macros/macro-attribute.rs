#![feature(unrestricted_attribute_tokens)]

#[doc = $not_there] //~ ERROR expected `]`, found `not_there`
fn main() { }
