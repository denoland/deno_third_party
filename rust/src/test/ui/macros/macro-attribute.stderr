error: expected `]`, found `not_there`
  --> $DIR/macro-attribute.rs:3:10
   |
LL | #[doc = $not_there] //~ ERROR expected `]`, found `not_there`
   |          ^^^^^^^^^ expected `]`

error: aborting due to previous error

